app/index.html

```diff
@@ -38,6 +38,22 @@
   <link integrity="" rel="stylesheet" href="{{rootURL}}assets/fe-redpencil.css">
 
   {{content-for "head-footer"}}
+
+  <!-- Matomo -->
+  <script type="text/javascript">
+    var _paq = window._paq = window._paq || [];
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+      var u="//dev-kaleidos-matomo.redpencil.io/";
```

Comment:
deze wordt geconfigureerd in ENV. Misschien ook best van daar sourcen?

Reply:
ok ik check het